pkg/systemd/manager.go

```diff
@@ -0,0 +1,132 @@
+package systemd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/pkg/errors"
+)
+
+const (
+	systemctl              = "systemctl"
+	systemctl_daemonreload = "daemon-reload"
+	systemctl_disable      = "disable"
+	systemctl_enable       = "enable"
+	systemctl_isactive     = "is-active"
+	systemctl_isenabled    = "is-enabled"
+	systemctl_start        = "start"
+	systemctl_status       = "status"
+	systemctl_stop         = "stop"
+
+	unitConfigurationBasePath_preferred   = "/lib/systemd/system"
```

Comment:
Question: is this the correct path to create the service file, since it seems to be created at runtime, rather than installed as a package.

(from the man pages)

       /lib/systemd/system  - System units installed by the distribution package  manager